test/test.js

```diff
@@ -3,6 +3,8 @@ var Dexie = require("dexie");
 var IDBExportImport = require("../index");
 var assert = require("assert");
 
+/* eslint-env mocha */
```

Comment:
Setting the environment removes some warnings relating to describe/it which would be unrecognised otherwise.

Reply:
What is mocha?

Reply:
Unit testing framework for JS: https://mochajs.org/